account_asset_management/migrations/12.0.1.1.3/end-migration.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Apps2GROW - Henrik Norlin
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+def update_asset_profile(env):
+    for profile in env['account.asset.profile'].search([]):
+        no_of_entries = profile.method_number_11
+        months_between_entries = profile.method_period_11
+
+        values = {}
+        # Number of years
+        values['method_number'] = int(no_of_entries * months_between_entries / 12)
```

Comment:
You can only do this if the division gives you integer number. If not, old method should be preserved (and you will need to install the other module).